### ISSUE TITLE
Change `.insectrc` location

### ISSUE THREAD

**Issue by mhmdanas** [sharkdp/insect#307]
Having `.insectrc` directly in `~` isn't super great in my opinion. Having it somewhere in `$XDG_CONFIG_HOME/insect` would be better.

**Comment by sharkdp** [comment]
I was recently looking for my insectrc in `~/.config`... so yeah. I totally agree. I don't think there is any disadvantage in also allowing ~/.insectrc (if only for backwards compatibility), but I would also suggest we switch the suggested default path to something in `$XDG_CONFIG_HOME/insect`. Maybe `$XDG_CONFIG_HOME/insect/init.ins`. Or just `$XDG_CONFIG_HOME/insect/init`.

**Comment by mhmdanas** [comment]
Which file should take priority if both are present, `~/.insectrc` or whatever new location we'll choose? I'm nclined to choose the latter.

**Comment by sharkdp** [comment]
Agreed

**Comment by sadmoonphoenix** [comment]
@mhmdanas I have a proposal. Basically after a lot of drama, developers developing the PolyMC project abandoned it and started Prism instead. Basically, once you open it for the first time, It'll tell you that there's config files, and tell you to move them. (Since it's only one file we can just have it moved by an insect tbh). 

TL;DR: I think the default location should be changed, but notify the user if they have a config file in ~/

**Comment by mhmdanas** [comment]
Thank you for your opinion.

> TL;DR: I think the default location should be changed, but notify the user if they have a config file in ~/

I'd rather not make this a breaking change that requires user intervention if this is what you mean, because the code needed to support the two locations is negligible. Just an informational message to inform the user might be okay, but I think even that isn't really necessary.

**Comment by sadmoonphoenix** [comment]
User intervention is certainly not necessary. The file should be moved, and the user should be notified as such. That's all.

**Comment by mhmdanas** [comment]
It absolutely would in some cases. For instance, if you manage your dotfiles in version control like I do, you'd have to move `.insectrc` manually in your dotfiles repo.
